README.md

```diff
@@ -31,13 +31,17 @@ Q2: What is the answer?
 ## Interview
 Hello!  In the rush that is the start of the term, I really haven't had a chance to chat with many of you.  So would you do me the favor of helping me get to know a bit about you by answering the following?  Please avoid one word answers... I want to learn about you!
 
-Q1: So you're finishing up your time in this CSCC program.  Why did you decide to embark on this journey?
+Q1: So you're finishing up your time in this CSCC program.  Why did you decide to embark on this journey?<br>
+*_Kate_: I'm currently a requirements analyst and have always wanted to become a developer. I was trying to learn on my own but it was overwhelming. I thought this would be the perfect opportunity.
 
-Q2: I _love_ to travel.  Tell me a bit about your favorite travel destination so far.  What's on your travel bucket list?
+Q2: I _love_ to travel.  Tell me a bit about your favorite travel destination so far.  What's on your travel bucket list?<br>
+*_Kate_: My favorite place so far has been NYC. I would love to travel to Italy at some point.
```

Comment:
Could you add some more about your NYC trip? What was one or two of the highlights?